content/en/docs/setup/windows/intro-windows-in-kubernetes.md

```diff
@@ -0,0 +1,400 @@
+:  Windows Server version 20H2
+
+The Kubernetes [version-skew policy](/docs/setup/release/version-skew-policy/) also applies.
+
+## Getting help and troubleshooting {#troubleshooting}
+
+Your main source of help for troubleshooting your Kubernetes cluster should start
+with the [Troubleshooting](/docs/tasks/debug-application-cluster/troubleshooting/)
+page.
+
+Some additional, Windows-specific troubleshooting help is referenced here. Logs are an 
+important element of troubleshooting issues in Kubernetes. Make sure to include them 
+any time you seek troubleshooting assistance from other contributors. Follow the 
+instructions in the SIG Windows 
+[contributing guide on gathering logs](https://github.com/kubernetes/community/blob/master/sig-windows/CONTRIBUTING.md#gathering-logs).
+
+* [Troubleshooting Windows Networking](/docs/setup/windows/windows-networking#troubleshooting-network)
+* [Node-level Troubleshooting](/docs/setup/windows/user-guide-windows-containers#troubleshooting-node)
+
+### Further investigation
+
+If these steps don't resolve your problem, you can get help running Windows containers on Windows nodes in Kubernetes through:
+
+* StackOverflow [Windows Server Container](https://stackoverflow.com/questions/tagged/windows-server-container) topic
+* Kubernetes Official Forum [discuss.kubernetes.io](https://discuss.kubernetes.io/)
+* Kubernetes Slack [#SIG-Windows Channel](https://kubernetes.slack.com/messages/sig-windows)
+
+### Reporting issues and feature requests
+
+If you have what looks like a bug, or you would like to
+make a feature request, please use the
+[GitHub issue tracking system](https://github.com/kubernetes/kubernetes/issues).
+You can open issues on
+[GitHub](https://github.com/kubernetes/kubernetes/issues/new/choose) and assign
+them to SIG-Windows. You should first search the list of issues in case it was
+reported previously and comment with your experience on the issue and add additional
+logs. SIG-Windows Slack is also a great avenue to get some initial support and
+troubleshooting ideas prior to creating a ticket.
+
+If filing a bug, please include detailed information about how to reproduce the problem, such as:
+
+* Kubernetes version: output from `kubectl version`
+* Environment details: Cloud provider, OS distro, networking choice and configuration, and Docker version
+* Detailed steps to reproduce the problem
+* [Relevant logs](https://github.com/kubernetes/community/blob/master/sig-windows/CONTRIBUTING.md#gathering-logs)
+
+It helps if you tag the issue as **sig/windows**, by commenting on the issue with `/sig windows`. This helps to bring
+the issue to a SIG Windows member's attention
+
+
+## {{% heading "whatsnext" %}}
+
+### Deployment tools
+
+The kubeadm tool helps you to deploy a Kubernetes cluster, providing the control
+plane to manage the cluster it, and nodes to run your workloads.
+[Adding Windows nodes](/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes/)
```

Comment:
Does this link need to change? “Adding Windows nodes” should probably move into the Setup section.